config/config.go

```diff
@@ -32,7 +32,11 @@ import (
 
 var cfg *ini.File
 
-func InitConfig(filename string) error {
+func InitConfig() {
```

Comment:
I Introduced it in order to be able to write functional tests which don't have config files